ranger-schema-registry-plugin/authorizer-shim/src/main/java/com/hortonworks/registries/ranger/authorization/schemaregistry/authorizer/Authorizer.java

```diff
@@ -0,0 +1,40 @@
+package com.hortonworks.registries.ranger.authorization.schemaregistry.authorizer;
+
+
+import java.util.Set;
+
+public interface Authorizer {
```

Comment:
Maybe we should move this to a different module. Authorizer class is agnostic to the underlying implement to putting it in ranger module may not make sense.

Reply:
Absolutely agree! Done!